make.py

```
@@ -1042,9 +1074,282 @@ def _delta_worker(data: tuple[str, str]) -> tuple[str, str, int, list[str]] | No
+                f"  name: Generate iso for {job_id}",
+                f"  needs: {job_id}",
+                "  uses: ./.github/workflows/iso.yaml",
+                "  secrets: inherit",
+                "  permissions: *permissions",
+                "  with:",
+                f"    variant: {job_id}",
+                "    pull: ${{ github.ref_name == 'master' && fromJson(needs.rootfs.outputs.updates) }}",
+                f"    offline: {json.dumps(offline)}",
+            ]
+
+        return indent(__(False) + [""] + __(True))
```

Ruff (0.14.4)

Consider `[*__(False), "", *__(True)]` instead of concatenation

Replace with `[*__(False), "", *__(True)]`

(RUF005)